### PR TITLE
Feature/grant access

### PR DESCRIPTION
**Pull request by andrewhick**
Made a number of changes to fit with latest UI changes.  Comments please :)

### CODE REVIEW

**Comment by andrewhick** [comment]
Still not sure why it's showing all my old commits since 19 Jan, given that they're already in master.

File: features/step_definitions/general_steps.rb [hunk 1]
```diff
@@ -53,7 +60,11 @@
 end
 
 Given(/^I select the "Name this licence" link$/) do
-  @front_app.licence_details_page.rename_link.click
+  if @front_app.licence_details_page.has_name_link?
```
Comment (timstone) [inline features/step_definitions/general_steps.rb]: I think you should move this conditional logic to the licence_details_page and create a method to call in your tests, that way if you have to do the same thing multiple times and that thing changes later on then you only have to change it once.

**Comment by timstone** [comment]
@andrewhick think we'll have to remove tidelift from the CI stack for the time being, come speak to Alan today about squashing your commit too.

File: features/step_definitions/general_steps.rb (hunk 1 above)
Comment (andrewhick) [reply features/step_definitions/general_steps.rb]: Done - see latest commit

**Pull request merged**